rpi_backlight.py

```diff
@@ -17,7 +17,9 @@
 __author__ = "Linus Groh"
 __version__ = "1.8.1"
 PATH = "/sys/class/backlight/rpi_backlight/"
-
+MODE = "MODE_RPI"
```

Comment:
That's wrong, should say `MODE = "RPI"`. Just put it below `MODE_RPI = "RPI"` like this:

```python
MODE_TINKERBOARD = "TINKERBOARD"
MODE_RPI = "RPI"

MODE = MODE_RPI
```

Reply:
Fixed